interop/src/server.rs

```diff
@@ -133,6 +135,14 @@ impl TestService for InteropTestService {
                     if let Some(param) = req.get_response_parameters().get(0) {
                         resp.set_payload(util::new_payload(param.get_size() as usize));
                     }
+                    // A Workaround for timeout_on_sleeping_server test.
+                    // The request only has 27182 bytes of zeros in payload.
```

Comment:
Is it still necessary?

Reply:
Yes.

Reply:
Why? You have set timeout to 1ms already at https://github.com/pingcap/grpc-rs/pull/203/files#diff-ef1f1e350c60784b68130f3cd88fbef6R170

Reply:
You can set it to 1ms, and you will find the client may receive a response.

Reply:
This is too hacky. And comment should talk about the problem this hack tries to work around.

Reply:
A friendly ping.